luigi/execution_summary.py

```diff
@@ -351,6 +353,8 @@ def _summary_format(set_tasks, worker):
     for status in _ORDERED_STATUSES:
         if status not in comments:
             continue
+        if status == 'ever_failed':
+            continue
```

Comment:
Actually, I'm pretty sure the previous if statement will always trigger before this. So just remove this if-statement please.